File: mainActivity/src/main/java/com/sierrawireless/avphone/MenuEntry.java
package com.sierrawireless.avphone;


enum MenuEntryType {
    TITLE,
    USER,
    COMMAND
}
public class MenuEntry {

    public String name;
    public  MenuEntryType type;
    MenuEntry(String name, MenuEntryType type) {
        this.name = name;
        this.type = type;
    }
}

File: mainActivity/src/main/java/com/sierrawireless/avphone/adapter/MenuEntry.kt
package com.sierrawireless.avphone.adapter

import android.graphics.drawable.Drawable

enum class MenuEntryType {
    TITLE,
    USER,
    COMMAND
}

class MenuEntry internal constructor(var name: String, var type: MenuEntryType, var drawable:Drawable? = null, var button:Boolean = false)
